Src/Core/WeaponController.cs

```diff
@@ -55,44 +51,32 @@ void Update()
                 swingAudioSource.Play();
             }
 
-            if(!swingBack) attackAngle = 3f * (weaponRenderer.flipY ? -1 : 1);
-            else attackAngle = -3f * (weaponRenderer.flipY ? -1 : 1);
+            attackAngle = swingBack ? -3f * (weaponRenderer.flipY ? -1 : 1) : 3f * (weaponRenderer.flipY ? -1 : 1);
 
             angle -= attackAngle;
 
             if(weaponRenderer.flipY){
-                if(angle >= prevAttackAngle + weapon.swift) swingBack = true;
-
-                if(angle <= prevAttackAngle){
-                    swingBack = false;
-                    attacking = false;
-                }
+                swingBack = (angle >= prevAttackAngle + weapon.swift);
```

Comment:
I only set it to true if the angle >= prevAttackAngle + weapon.shift, but i DON'T set it to false otherwise, it gets set to false in other statements

Reply:
Same applies to line 62